torchaudio/csrc/lfilter.cpp

```diff
@@ -1,5 +1,6 @@
 #include <torch/script.h>
 #include <torch/torch.h>
+#include <torchaudio/csrc/iir_cuda.h>
```

Comment:
@yoyololicon Could you also apply `#ifdef USE_CUDA` here?

Some of the builds with more strict complier setting are failing with 

`Stderr: ld.lld: error: undefined symbol: cuda_lfilter_core_loop(at::Tensor const&, at::Tensor const&, at::Tensor&)`

Reply:
I see. I just commited it, did that work?

Reply:
let me try again.

Reply:
Actually it was something else. Things should be good. Will try to land this by tomorrow.